language/move-stdlib/modules/addresses.move

```diff
@@ -0,0 +1,5 @@
+// Declaration of Std address. Needs to be assigned a value depending on the deployment
+
+// TODO the tooling around this tries to actually build the modules... which isn't great
+// This value should be removed
+address Std = 0x1;
```

Comment:
@vgao1996 the whole tooling failed since I think the stdlib tries to build itself as a binary? This isn't something we can really support anymore in the named address land since it needs to be configured per deployment. Not sure what to do about this going forward

Reply:
So just to clarify, going forward, how are the addresses supposed to be bound? I mean I know that you can put the assignment in a .move file, but I feel like you might also need to make them specifiable via the movec interface.

A related but somewhat orthogonal issue is the doc generator. I'm not sure if they can handle named addresses. Have you already added named address support to it?